src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
The implementation of ``UnhandledExceptionHandlerUnix`` in ``exceptionhandling.cpp`` is available only when ``WIN64EXCEPTIONS`` is defined, but ``FuncEvalHijack``in ``debug/ee/i386/dbghelpers.S`` uses ``UnhandledExceptionHandlerUnix`` as a personality routine even when ``WIN64EXCEPTIONS`` is not defined (which causes undefined reference issue in the checked-in stuff).

We could eliminate the above stub once we enable ``WIN64EXCEPTIONS``.